Implement the colorful power chart

xopok opened the pull request:
![2](https://user-images.githubusercontent.com/4334/82740796-58233880-9d4c-11ea-829e-c7a083218b57.png)

xopok committed:
Add PowerFieldWatchLayout

xopok committed:
Use PowerFieldWatchLayout for round and semi-round screen shape.

This might not cover all possible watch screens (e.g. FR920 does have a rectangular one) but for now it's the easiest I could come up with.

xopok committed:
Add PowerFieldWideWatchLayout

xopok committed:
Implement power chart

xopok committed:
Add the "Use alternative font over power chart" property

Also prepend @Strings. to titles, the opportunity for i18n.

xopok committed:
Add the Edge layout for power chart support. Tweak font sizes.

xopok committed:
Make the chart drawing compatible with different layouts and datafield sizes.

Only draws the chart if for:
- Edge devices and square watches:
  - the datafield height is at least 1/3 of the screen height.
- Round and semi-round watches:
  - the datafield is the middle one in the 3A layout (then also displays the cadence).

Alternative font is now an option only for layouts allowing the chart.

xopok committed:
Use system fg/bg colors for alt font.

xopok committed:
Don't use alt font on watches

(hot fix)

axl13 commented on resources/resources.xml:
```diff
@@ -1,18 +1,48 @@
-		  <label id="value" x="center" y="center"
-				font="Graphics.FONT_SYSTEM_NUMBER_THAI_HOT"
-				justification="Graphics.TEXT_JUSTIFY_CENTER"/>
-		  <label id="mark" x="center" y="top"
```
Why was this removed?

axl13 commented on source/PowerAdjusterView.mc:
```diff
@@ -266,43 +277,36 @@ class PowerDataField extends Ui.DataField {
-    setLayout(Rez.Layouts.PowerFieldLayout(dc));
-  }
-
-  function drawZones(dc, zone, power) {
```
I'd prefer to keep the old functionality with the color indicator (it's easier to see which zone you in the interval), on top on ability to change to the Graph (in options) if one prefers.

axl13 reviewed (commented):
Please keep the old way of coloring zones in the options. Thanks!

xopok committed:
Update label font sizes for Edge devices to match the actual size

xopok committed:
Add fenix6 family to the manifest file. (#21)

Co-authored-by: Oleksiy Mishchenko <amishchenko@gmail.com>

xopok committed:
Merge pull request #22 from axl13/master

sync

xopok committed:
Fix orphan "-" when in alt font mode.

axl13 committed:
Update PowerAdjusterView.mc

Otherwise `dc_hight.toFloach` gonna checkout at line 163, if Layout is not initialized (which is the case on Edge)
